packages/menus/src/containers/MenuContainer.js

```diff
@@ -499,97 +499,96 @@ class MenuContainer extends ControlledComponent {
               );
             }}
           </Target>
-          <Popper
-            placement={this.convertGardenToPopperPlacement()}
-            eventsEnabled={eventsEnabled}
-            modifiers={popperModifiers}
-          >
-            {({ popperProps, scheduleUpdate }) => {
-              const popperPlacement = popperProps['data-placement'];
-              const outOfBoundaries = popperProps['data-x-out-of-boundaries'];
-
-              if (!isOpen) {
-                return false;
-              }
-
-              const menu = (
-                <MenuWrapper innerRef={popperProps.ref} style={popperProps.style} zIndex={zIndex}>
-                  <FocusJailContainer focusOnMount={false}>
-                    {({ getContainerProps: getFocusJailContainerProps, containerRef }) => (
-                      <SelectionContainer
-                        id={id}
-                        direction="vertical"
-                        focusedKey={controlledFocusedKey}
-                        selectedKey={selectedKey}
-                        defaultFocusedIndex={defaultFocusedIndex}
-                        onStateChange={newState => {
-                          /**
-                           * We only care if `selectedKey` is involved with the state change
-                           */
-                          if (Object.prototype.hasOwnProperty.call(newState, 'selectedKey')) {
-                            let newSelectedKey = newState.selectedKey;
-
-                            if (newSelectedKey === undefined) {
-                              newSelectedKey = selectedKey;
+          {isOpen && (
```

Comment:
I believe this is the only change? 😂

Reply:
Yep the rest was prettier reformatting, best viewed with `?w=1`

Reply:
💯 thanks, Ryan~